### DISCUSSION TITLE
OMV OneDrive plugin - error AADSTS900144: The request body must contain the following parameter: 'scope'.

### DISCUSSION THREAD

**Discussion by ilpratti** [abraunegg/onedrive#2928] [Q&A] [answered]
dears, I installed the onedrive plugin in OMV.
following the instruction I launched on terminal the command omv-onedrive. It generated a link and when I clik on it I got an erron in the web page: AADSTS900144: The request body must contain the following parameter: 'scope'.

Am I doing something wrong?
Thanks

**Comment by ilpratti** [comment]
<img width="824" alt="image" src="https://github.com/user-attachments/assets/2a481faf-9d1a-4040-b15f-7c02400a5b89">

Note: when I installed the plugin I got a strange message "connection lost" instead of "END OF LINE". Could it be connected to my issue?

**Comment by abraunegg** [comment, marked as answer]
@ilpratti 
> OMV OneDrive plugin

I do not support the OMV OneDrive Plugin - please raise all OMV questions here: https://forum.openmediavault.org/

> It generated a link and when I clik on it I got an erron in the web page: AADSTS900144: The request body must contain the following parameter: 'scope'.

You need to authenticate the client in the correct manner. This error indicates there is an issue with:
* Your account
* The way you are authenticating 
* The permissions you have to authenticate the client

This is nothing that can be solved here. Please raise a support ticket with whoever supports your Microsoft 365 environment.

**Comment by abraunegg** [comment]
> It generated a link and when I clik on it I got an erron in the web page: AADSTS900144: The request body must contain the following parameter: 'scope'.

For future readers, the solution for this odd problem was not to use Chrome but Microsoft Edge to authenticate the client.

I am unsure why this was the case, as I use Chrome exclusively and never run into authentication issues like this.
